src/components/PropertyProduct.js

```diff
@@ -28,6 +27,7 @@ class PropertyProduct extends Component {
                 <img alt={product.device_name} src={product.image} width="60%" />
               </div>
               <div className="product-info">
+                {product.shdb_url && <a href={product.shdb_url} target="_blank">View on SmartHomeDB &raquo;</a>}
```

Comment:
There are several properties that have a null url, please verify url before rendering a link.